commands/local_server_start.go

```diff
@@ -274,11 +275,26 @@ var localServerStartCmd = &console.Command{
 		if p.PHPServer != nil {
 			msg += fmt.Sprintf("     The Web server is using %s %s\n", p.PHPServer.Version.ServerTypeName(), p.PHPServer.Version.Version)
 		}
-		msg += fmt.Sprintf("\n     <href=%s://127.0.0.1:%d>%s://127.0.0.1:%d</>", scheme, port, scheme, port)
+		applicationUrl := fmt.Sprintf("%s://127.0.0.1:%d", scheme, port)
+		msg += fmt.Sprintf("\n     <href=%s>%s</>", applicationUrl, applicationUrl)
 		if proxyConf, err := proxy.Load(homeDir); err == nil {
 			for _, domain := range proxyConf.GetDomains(projectDir) {
 				msg += fmt.Sprintf("\n     <href=%s://%s>%s://%s</>", scheme, domain, scheme, domain)
 			}
+
+			domains := proxyConf.GetReachableDomains(projectDir)
+			if len(domains) > 0 {
+				applicationUrl = fmt.Sprintf("%s://%s", scheme, domains[0])
+			}
+		}
+
+		if config.Open {
+			if err := open.Run(applicationUrl); err != nil {
+				terminal.Eprintln("<error>Error while opening:", err, "</>")
+				terminal.Eprintfln("Please visit <href=%s>%s</> manually.", applicationUrl, applicationUrl)
+			} else {
+				terminal.Eprintfln("Opened: <href=%s>%s</>", applicationUrl, applicationUrl)
+			}
```

Comment:
```suggestion
			abstractOpenCmd(applicationUrl);
```

should be enough. or did I miss something?

Reply:
Indeed it works, fixed